Cron/KlSyncs.php

```diff
@@ -146,7 +146,6 @@ private function sendUpdatesToApp(array $groupedRows, bool $isRetry = false)
                         $storeId = '';
                         if (isset($decodedPayload['StoreId'])) {
                             $storeId = $decodedPayload['StoreId'];
-                            unset($decodedPayload['StoreId']);
```

Comment:
Adding this to the payload because we may want to use it down the line for variant triggered flows in the app